test/integration/internal-extension.js

```diff
@@ -83,21 +83,24 @@ test('load sync', t => {
     t.ok(vm.extensionManager.isExtensionLoaded('coreExample'));
 
     t.equal(vm.runtime._blockInfo.length, 1);
-    t.equal(vm.runtime._blockInfo[0].blocks.length, 1);
+    t.equal(vm.runtime._blockInfo[0].blocks.length, 2);
```

Comment:
Maybe add a comment here that there are 2 items but only one actual block. The other one is a button.